open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -13,10 +13,25 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def get_Onecolumn_groupby_sum(df,columnname):
</pre><p>Please follow the <a href="https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/" target="_blank">snake style</a> to name the function  as <code>def groupby_and_sum(df, column_name)</code></p><p> </p><p>And please not use nested if/else here. You can do it as below.</p><pre class="ql-syntax" spellcheck="false">if df is None:
  raise Exception('df cannot be None.')

if column_name not in df:
  raise Exception('Column does not exist.')

eturn df.______(column_name).______
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Modified</p><p> </p>